src/WebJobs.Script/Host/IProxyClientGenerator.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public interface IProxyClientGenerator
+    {
+        /// <summary>
+        /// The return type is object to facilitate dynamic loading of Proxy dlls
+        /// </summary>
+        object CreateProxyClient(string proxyJson, ILogger logger);
```

Comment:
Removed the Class as we can use the IProxyClient in the ScriptHost now that we are fine with loading the shim.

---
In reply to: [132560257](https://github.com/Azure/azure-webjobs-sdk-script/pull/1749#discussion_r132560257) [](ancestors = 132560257)